ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/HardFork/Combinator/Ledger.hs

```diff
@@ -1044,19 +1045,21 @@ distribLedgerTables ::
   -> LedgerTables (LedgerState                x  ) mk
 distribLedgerTables idx =
     LedgerTables
-  . mapKeysMK distrTxIn
-  . mapMaybeMK distrTxOut
+  . mapKeysMK (distribCanonicalTxIn idx)
+  . mapMaybeMK (distribTxOut idx)
   . getLedgerTables
-  where
-    distrTxIn :: Key (LedgerState (HardForkBlock xs)) -> Key (LedgerState x)
-    distrTxIn = distribCanonicalTxIn idx
```

Comment:
Maybe keep this and expose it like `distribTxOut`

Reply:
I think it is fine like it is now, making advantage that it is Canonical